app/livechat/server/lib/routing/LeastAmount.js

```diff
@@ -0,0 +1,34 @@
+import { RoutingManager } from '../RoutingManager';
+import { LivechatDepartmentAgents, Users } from '../../../../models';
+
+/* Least Amount Queuing method:
+	*
+	* default method where the agent with the least number
+	* of open chats is paired with the incoming livechat
+*/
+class LeastAmount {
+	constructor() {
+		this.config = {
+			previewRoom: false,
+			showConnecting: false,
+			showQueue: false,
```

Comment:
according to [this previous code](https://github.com/RocketChat/Rocket.Chat/pull/15003/files#diff-35ad4c8f424bcf2b344f149d439212d4L91) the correct value should be `true`